### ISSUE TITLE
Nginx module

### ISSUE THREAD

**Issue by ghost** [TextGlass/c#5]
Hello!

Found this project while looking at dClass (https://github.com/TheWeatherChannel/dClass).
Can TextGlass be a replacement for dClass as a Nginx module, or doesn't TextGlass work with Nginx?

**Comment by rezan** [comment]
Yes, textglass is the replacement for dclass, the problem is that we do not have a nginx module at the moment. Is this something you could contribute?
